qemu/tests/cfg/qmp_event_notification.cfg

```diff
@@ -10,37 +10,51 @@
     monitor_type = human
     monitor_type_qmp2 = qmp
     variants:
+        - from_guest:
+            only qmp_system_reset qmp_system_powerdown qmp_rtc_change qmp_watchdog
+            event_cmd_type = guest_cmd
+        - from_qmp:
+            only qmp_quit qmp_system_reset qmp_system_powerdown
+            event_cmd_type = qmp_cmd
+        - from_hmp:
+            only qmp_stop qmp_resume
+            event_cmd_type = monitor_cmd
+    variants:
         - qmp_quit:
             event_cmd = quit
-            event_cmd_type = qmp_cmd
             event_check = "SHUTDOWN"
         - qmp_system_reset:
-            event_cmd = system_reset
-            event_cmd_type = qmp_cmd
             event_check = "RESET"
+            from_guest:
+                event_cmd = reboot
+                Windows:
+                    event_cmd = shutdown /r /f /t 0
+            from_qmp:
+                event_cmd = system_reset
         - qmp_system_powerdown:
-            event_cmd = system_powerdown
-            event_cmd_type = qmp_cmd
-            event_check = "POWERDOWN"
+            from_guest:
+                event_check = "SHUTDOWN"
+                event_cmd = shutdown -h now
+                Windows:
+                    event_cmd = shutdown /s /f /t 0
```

Comment:
so pls put Windows cmd to guest-os.

if you have better idea for the format of configure file, pls open a topic to discuss it.